docs/api/structures/window-state-restore-options.md

```diff
@@ -0,0 +1,3 @@
+# WindowStateRestoreOptions Object
+
+* `stateId` string - A unique identifier used for saving and restoring window state.
```

Comment:
Can we make this something more friendly, don't change immediately this is meant to start a discussion thread here.

I'd be in favour of just a generic `BaseWindow` constructor option for a window "name" which we can document as a unique identifier that apps can use to identify windows, and Electron can use internally for things like this state persistance.

Then change this options object to be just `enabled: true/false`.

Apologies if this option has already been discussed or bikeshed elsewhere.

Reply:
@erickzhao interested in your opinions here